package.json

```diff
@@ -71,5 +71,22 @@
   },
   "dependencies": {
     "jasmine": "^5.1.0"
+  },
+  "resolutions": {
+    "victory-area": "36.8.6",
+    "victory-bar": "36.8.6",
+    "victory-brush-container": "36.8.6",
+    "victory-core": "36.8.6",
+    "victory-cursor-container": "36.8.6",
+    "victory-create-container": "36.8.6",
+    "victory-legend": "36.8.6",
+    "victory-line": "36.8.6",
+    "victory-pie": "36.8.6",
+    "victory-scatter": "36.8.6",
+    "victory-selection-container": "36.8.6",
+    "victory-shared-events": "36.8.6",
+    "victory-tooltip": "36.8.6",
+    "victory-voronoi-container": "36.8.6",
+    "victory-zoom-container": "36.8.6"
```

Comment:
Sure it makes sense, but for information, can you just clarify why these are included?